### PR TITLE
Popup branch

### PR DESCRIPTION
**Pull request by frnega**
in this pull request added popup window both for mobile and desktop

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -31,8 +142,208 @@ document
...
+document
+  .getElementsByClassName('cancel-6')[0]
+  .addEventListener('click', closePopup6);
+
+const storeObj = {
+  1: {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    featuredImg: [
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
```
Comment (tufoinnkuo10) [inline menu.js:194]: - [ ] great job. please kindly align the heading and techs to the left like in the figma version.
yours
Screenshot (528)_LI
figma
Screenshot (526)_LI

File: menu.js [hunk 2]
```diff
@@ -31,8 +142,208 @@ document
...
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
```
Comment (tufoinnkuo10) [inline menu.js:196]: - [ ] kindly centralize your bottom text to follow the figma design. great job 
yours 
Screenshot (529)_LI
figma
Screenshot (527)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @frnega 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**